docs/libcurl/opts/CURLOPT_SSL_OPTIONS.3

```diff
@@ -55,6 +55,14 @@ offline distribution points for those SSL backends where such behavior is
 present. This option is only supported for Schannel (the native Windows SSL
 library). If combined with \fICURLSSLOPT_NO_REVOKE\fP, the latter takes
 precedence. (Added in 7.70.0)
+
+.IP CURLSSLOPT_NATIVE_CA
+With the build against OpenSSL library, uses the native operating system
+store of root certificates instead of any other CA certificates.
+Currenly supported with MS-Windows.
+Caution: use only CURLOPT_CAINFO with curl 7.70.1 or later for MS-Windows
+using OpenSSL library. On other configuration, curl will have no certificate.
```

Comment:
This sentence has a weird word oder.

Reply:
can you suggest me alternative text modified?
My english is not perfect, so I prefer you suggest a new version than ask me fix, then review and ask modify again

Reply:
Will do

Reply:
I don't understand the sentence at all. It describes `CURLSSLOPT_NATIVE_CA` and then it speaks of `CURLOPT_CAINFO` without explaining how they relate here. Can you elaborate what this is trying to say?

Reply:
My edited version says this now, but I'd like to understand the code about `CURLOPT_CAINFO` so we can perhaps add that:
~~~
Tell libcurl to use the operating system's native CA store for certificate
verifiction. Works when built to use OpenSSL on Windows. (Added in 7.71.0)

Reply:
wa must remove CURLOPT_CAINFO reference. I wrote this test before we add CURLSSLOPT_NATIVE_CA

So now, we must only say CURLOPT_CAINFO is only for new version of curl (for windows , and pehaps on other OS later).
Before new version of curl, we need provide a file of certificate

Reply:
We can possibly add that `CURLSSLOPT_NATIVE_CA` overrides `CURLOPT_CAINFO` if both are set...